torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -72,11 +72,14 @@ struct TORCH_API KinetoThreadLocalState : public ProfilerThreadLocalState {
           .fwdThreadId(ctx->fwdThreadId)
           .scope(ctx->recFunScope);
       if (ctx->shapes && !ctx->shapes->empty()) {
-          kineto_events_.back().shapes(*ctx->shapes);
+        kineto_events_.back().shapes(*ctx->shapes);
       }
       if (ctx->stack && !ctx->stack->empty()) {
         kineto_events_.back().stack(*ctx->stack);
       }
+      if (ctx->extraArgs && !ctx->extraArgs->empty()) {
+        kineto_events_.back().flops(computeFlops(std::string(fn.name().str()), *ctx->extraArgs));
```

Comment:
in case computeFlops cannot compute flops it returns zero, will we see these zeroes in the profiler output?

Reply:
(in this case, let's remove them from the profiler output)

Reply:
> in case computeFlops cannot compute flops it returns zero, will we see these zeroes in the profiler output?

No, the profiler.py will automatically ignore zero values so the user will not see them: https://github.com/pytorch/pytorch/blob/master/torch/autograd/profiler.py#L1536